tests/test_unittest/test_gtc/test_gtir.py

```diff
@@ -19,7 +17,7 @@
     VerticalLoop,
 )
 
-from .gtir_utils import DummyExpr, FieldAccessBuilder
+from .gtir_utils import DummyExpr, FieldAccessBuilder, ParAssignStmtBuilder, StencilBuilder
```

Comment:
There is already a `class gt4py.stencil_builder.StencilBuilder`, but I would agree this shoudn't lead to any confusion here,